src/Donations/Micros/PlanetCashSelector.tsx

```diff
@@ -61,6 +63,15 @@ const PlanetCashSelector: FC = (props) => {
     }
   }, [isPlanetCashActive, setcountry]);
 
+  useEffect(() => {
+    const countryData = countriesData.filter(
+      (singleCountryData) =>
+        singleCountryData.countryCode == profile?.planetCash.country
+    );
+    setplanetCashCountry(countryData[0].countryName);
+    console.log(countryData);
+  }, [profile]);
```

Comment:
1. There is a util function `getCountryDataBy` that can map country code to country name. Perhaps some of this code isn't needed.
2. In the `useEffect` above this, we are checking if planet cash is active, and setting country. Perhaps that would be a better place to set the planet cash country name, if we need to set it?